testscript/doc.go

```diff
@@ -120,6 +120,35 @@ when testing.Short() is false.
 
 Additional conditions can be added by passing a function to Params.Condition.
 
+An example:
```

Comment:
No need for this to be a separate paragraph, IMO. You can attach it to the previous paragraph.

Reply:
removed